src/pykeen/trackers/tensorboard.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+"""An adapter for TensorBoard."""
+import datetime
+import pathlib
+from typing import Any, Dict, Mapping, Optional
+
+from .base import ResultTracker
+from ..constants import PYKEEN_LOGS
+from ..utils import flatten_dictionary
+
+__all__ = [
+    'TensorBoardResultTracker',
+]
+
+
+class TensorBoardResultTracker(ResultTracker):
+    """A tracker for TensorBoard."""
+
+    def __init__(
+        self,
+        experiment_path: Optional[str] = None,
+        experiment_name: Optional[str] = None,
+        tags: Optional[Dict[str, Any]] = None,
+    ):
+        """
+        Initialize result tracking via Tensorboard.
+
+        :param experiment_path:
+            The experiment path. A custom path at which the tensorboard logs will be saved.
+        :param experiment_name:
+            The name of the experiment, will be used as a sub directory name for the logging. If no default is given,
+            the current time is used. If set, experiment_path is set, this argument has no effect.
+        :param tags:
+            The additional run details which are presented as tags to be logged
+        """
+        from torch.utils.tensorboard import SummaryWriter as _SummaryWriter
+        self.SummaryWriter = _SummaryWriter
+        self.tags = tags
+
+        if experiment_path is None:
+            if experiment_name is None:
+                experiment_name = datetime.datetime.now().isoformat()
+            path = PYKEEN_LOGS / f"tensorboard/{experiment_name}"
```

Comment:
That forward slash in the path part will be an issue on windows computers. Try one of the following instead:

```suggestion
            path = PYKEEN_LOGS / "tensorboard" / experiment_name
```
OR
```suggestion
            path = PYKEEN_LOGS.joinpath("tensorboard", experiment_name)
```